src/TextParseNode.cs

```diff
@@ -66,15 +66,15 @@ public TextParseNode(string? text)
     /// <inheritdoc />
     public Time? GetTimeValue() => DateTime.TryParse(Text, out var result) ? new Time(result) : null;
     /// <inheritdoc />
-    #if NET5_0_OR_GREATER
-    public IEnumerable<T?> GetCollectionOfEnumValues<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] T>() where T : struct, Enum
+#if NET5_0_OR_GREATER
+    public IEnumerable<T?> GetCollectionOfEnumValues<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] T>() where T : struct, Enum
```

Comment:
@IEvangelist just made the opposite change in #138.
Could we have a discussion here before making further changes please?

Reply:
It looks like `Microsoft.Kiota.Abstractions` is out of date, meaning the version this is built against currently has `DynamicallyAccessedMemberTypes.All` set on `WriteCollectionOfEnumValues`. I can wait until the abstractions library is version bumped and rebase to remove this change.

Reply:
This is why I believe these repositories should be consolidated into one, all of the kiota abstractions are in their own repo which make updates VERY tedious and error prone. If they were in the same repo, this wouldn't be the case.

Reply:
It should be a bit less error prone now trimming issues will be caught at compile time, but yeah, I'd agree with that.

Reply:
feedback noted, this is something we could consider since java and typescript are on a consolidated repo.
In the meantime, I believe this change still need to be reverted, correct?

Reply:
Ah, yes, sorry, reverted in https://github.com/microsoft/kiota-serialization-text-dotnet/commit/e752f6c4ec2e3191cd8824ccb207727d4579d6b1